WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerActivity.kt

```diff
@@ -499,6 +499,10 @@ class StoryComposerActivity : ComposeLoopFrameActivity(),
                 }
             }
 
+            viewModel.onStorySaved()
+            // TODO add tracks
+            processStorySaving()
```

Comment:
I moved this call up here because we need the AddedViews to be read from PhotoEditor (in case they have just finished editing stuff on the last Story frame that was being shown on the Story composer) and added to the StoryRepository properly.
(see inside `processStorySaving()` --> `addCurrentViewsToFrameAtIndex()`)